packages/wrangler/src/config/index.ts

```diff
@@ -374,7 +375,7 @@ export function printBindings(
 				}
 				return {
 					key: binding,
-					value: bucket_name,
+					value: `${bucket_name}${local ? " (local)" : ""}`,
```

Comment:
A helper function would clean up this file a bit:

```
value: addLocalSuffix(bucket_name, local)
```

Reply:
https://github.com/cloudflare/workers-sdk/pull/6972/commits/5bc416c131992dd5494bdec5b9ded819dd979b98